examples/jsm/nodes/shadernode/ShaderNode.js

```diff
@@ -34,7 +38,11 @@ const shaderNodeHandler = {
 
 		if ( typeof prop === 'string' && node[ prop ] === undefined ) {
 
-			if ( NodeElements.has( prop ) ) {
+			if ( node.isStackNode !== true && prop === 'assign' ) {
+
+				return ( ...params ) => assign( node, ...params );
```

Comment:
```suggestion
				return ( ...params ) => currentStack.assign( node, ...params );
```

Maybe we shouldn't export this "modified" assign, so we can just inline it here?